src/db/models/data_repo_commits/mod.rs

```diff
@@ -30,6 +30,9 @@ pub struct DataRepoCommits {
     pub auth_commit_timestamp: String,
     /// Foreign key reference to the publication.
     pub publication_id: String,
+    /// Type of commit: 0 for build date, 1 for codified date.
+    #[serde(default)]
+    pub commit_type: i64,
```

Comment:
I'm a bit confused by naming here, just to clarify: `build date` and `codified date` are two types of dates we can have. What is commit that has type `build date`?
- a commit that doesn't have `codified date` but just `build date`?

Is the reason for this change to know what `date` is?